gate-core/src/main/groovy/com/netflix/spinnaker/gate/services/internal/Front50Service.groovy

```diff
@@ -143,4 +143,13 @@ interface Front50Service {
 
   @GET('/serviceAccounts')
   List<ServiceAccount> getServiceAccounts()
+
+  //
+  // Delivery-related
+  //
+  @GET('/deliveries')
+  List<Map> getDeliveries()
```

Comment:
this is a ~50 line PR so I have license to be pedantic 😉 — for whatever reason `deliveries` sounds more like an execution of a delivery flow than the config for a delivery flow. Any interest in something like `deliveryConfigs` to match `pipelineConfigs`?

Reply:
Sure, sounds good.

Reply:
I renamed the general endpoint in gate. However, I'm going to keep the front50 endpoint the same (deliveries) because that's how it is for application configs. In front50, when you get `applications`, it's all the application configs.

Reply:
:+1: I'm into it